### PR TITLE
Resolves #8 by providing documentation link and simple Makefile

### LINKED ISSUE #8: Wanted: More detailed deployment instructions
Hi

I am trying to get the hello-world anaconda-addon to work. I have created the CPIO image and I am sending it via http. The kernel-cmdline includes inst.updates=http:/192.168.X.X/update.img.
The kickstart file (that is also supplied via http) contains 
%addon org_fedora_hello_world
Random Text
%end

But still, it doesn't do anything! (Meaning: it doesn't show up in the anaconda GUI).
What am I supposed to see? Why does glade crash when I open the ui file?
Any hint is appreciated.
I studied http://rhinstaller.github.io/anaconda-addon-development-guide/index.html
but to no avail.

### PR DESCRIPTION
**Pull request by jcpunk**
This PR should make modifying and playing with the addon a bit simpler for new folks.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+TMPDIR := $(shell mktemp -d)
+OUTDIR := $(shell pwd)
+
+_default:
+	@mkdir -p $(TMPDIR)/usr/share/anaconda/addons/
+	@cp -par org_fedora_hello_world $(TMPDIR)/usr/share/anaconda/addons/
+
```
Comment (jkonecny12) [inline Makefile]: Please remove these leading whitespace characters.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+TMPDIR := $(shell mktemp -d)
+OUTDIR := $(shell pwd)
+
+_default:
+	@mkdir -p $(TMPDIR)/usr/share/anaconda/addons/
+	@cp -par org_fedora_hello_world $(TMPDIR)/usr/share/anaconda/addons/
+	
+	@cd $(TMPDIR)
+	@find . | cpio -c -o --quiet | gzip -9 > $(OUTDIR)/hello_world_addon_updates.img
```
Comment (jkonecny12) [inline Makefile]: Please change the find part in `find .` to `find ./org_fedora_hello_world`.

File: Makefile (hunk 1 above)
Comment (jcpunk) [reply Makefile]: I'm not sure I understand this request.  If I remove the tabs, the Makefile syntax is incorrect.

File: Makefile (hunk 2 above)
Comment (jcpunk) [reply Makefile]: I pulled this find from[1].  I'll make it more precise, but I believe it does need to preserve the directory structure.

[1] https://rhinstaller.github.io/anaconda-addon-development-guide/sect-anaconda-addon-deploying-testing.html

File: Makefile (hunk 1 above)
Comment (jkonecny12) [reply Makefile]: I mean remove tabs (indentation) on the empty lines. That won't break Makefile it should work correctly.

File: Makefile (hunk 2 above)
Comment (jkonecny12) [reply Makefile]: Ohh, I should have explained the problem. The point is that when you do `cp -par` to temp and then `find .` you will also add to the image other files which shouldn't be there (whole .git or IDE folders for example). 

You can use lsinitrd tool to inspect the image but inside of image should be *only* folder with addon which is `org_fedora_hello_world` in this case.

Maybe that find is not the correct solution but the `.git` folder really shouldn't be there.

**Comment by jcpunk** [comment]
I believe I've got the requested fixes in place

**Pull request merged**